docs/navigate.rst

```diff
@@ -266,3 +274,9 @@ It displays the **default route domain** for the partition to which the Virtual
 If you encounter this issue, use the 'zoom out' option on your web browser. For example, on Windows, hold the **Ctrl** key, and then click **-** (the dash/minus key).
 
 ------
+
+**Q. VIP MAC masquerade Address might display old MAC on the visibility dashboard**
+
+Due to APIC endpoint learning and retention behavior, it takes 10-15 mins time for updated MAC details to be available on APIC endpoint tracker. Until APIC updates the MAC address, FASC visibility dashboard displays the old MAC address. Refer this link for recommended actions:  https://support.f5.com/csp/article/K44023455
```

Comment:
Due to the APIC endpoint learning and retention behavior, it takes 10-15 minutes for the updated MAC details to be available on the APIC endpoint tracker.
Until APIC updates the MAC address, the FASC visibility...